### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by davidgasquez**
Couple of small typo changes after running through the docs.

### CODE REVIEW

File: src/metrics.md [hunk 1]
```diff
@@ -24,34 +24,34 @@ The following abbreviations are used throughout this page and site:
...
-This shows the number of Storage Providers in a given day that met the ["cold" service class service level objectives](https://github.com/filecoin-project/service-classes/service-classes/cold.md).  If a given SP only meets some of the SLOs, these are highlighted as well. 
+This shows the number of Storage Providers in a given day that met the ["cold" service class service level objectives](https://github.com/filecoin-project/service-classes/service-classes/cold.md).  If a given SP only meets some of the SLOs, these are highlighted as well.
 
 ## Individual Storage Provider Metrics
 These are metrics that specific to an individual SP at `/provider/providerId`.
 
 ### Spark Retrieval Success Rate
-This shows a Storage Provider's [Spark Retrieval Success Rate](https://github.com/filecoin-project/service-classes/service-level-indicators/spark-retrieval-success-rate.md).  The horizontal line's value, which corresponds with a service level objective, is defined in the supporting [service class definition](https://github.com/filecoin-project/service-classes/service-classes) (e.g., ["warm"](https://github.com/filecoin-project/service-classes/service-classes/warm.md)).
```
Comment (BigLep) [inline src/metrics.md:45]: I think "e.g.," is right.  https://www.scribbr.com/definitions/for-example-abbreviation/

**Review by BigLep — APPROVED** [review]
Thanks for cleaning up all my typos.  Is there a suggested extension to help do do spelling/grammar checks when doing markdown editing?

**Comment by davidgasquez** [comment]
Been using [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker). You'll have to tweak it a little, as by default is very annoying (it'll lint code).

File: src/metrics.md (hunk 1 above)
Comment (davidgasquez) [reply src/metrics.md:45]: Interesting! TIL. Will update the linter rules :sweat_smile:

**Pull request merged**